src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -3145,12 +3279,53 @@ mesh::topology::unstructured::verify(const Node &topo,
             {
                 elems_res &= verify_integer_field(protocol, topo_elems, info_elems, "offsets");
             }
```

Comment:
The "offsets" field has been rolled into the one-to-many relations construct and thus no longer needs to be separately checked here.

Reply:
I think this "offsets" check here is fine, for the case where a non-polygonal/non-polyhedral shape has the offsets optionally generated.

Reply:
My understanding was that we were going to get rid of the "offsets" field for non-polygonal/polyhedral topologies since it was redundant and could consequently cause inconsistencies (e.g. a "quad" topology with an "offsets" array of `[0, 4, 8, 11, ...]`. This is actually why I removed the "topology/connectivity/offsets" section from [the documentation](https://github.com/LLNL/conduit/pull/579/files#diff-2aaf2cc46f2369eddb166bf2d13ed654L414).

@cyrush: I'm curious to hear your thoughts since my understanding was informed by a discussion between you and @agcapps about [the interaction between "shape" and "sizes"](https://github.com/LLNL/conduit/issues/511#issuecomment-597341371). The outcome of that discussion seemed to be that some "shape" specifiers made "sizes"  redundant, so they shouldn't be used in those cases. Given that it's a derived/related quantity, do we want to keep the "offsets" field for potentially faster indexing on static-shaped topologies (e.g. "tris", "quads")?

Reply:
`shape` vs `sizes` isn't good b/c they could both be present and conflict. But `offsets` could be useful in the non poly case, b/c it would allow folks to zero copy via an indirection array.  Blueprint consumers will need to be upgraded to support this -- but it seems like a good thing?

Reply:
I agree that greater flexibility is generally desirable, particularly in cases like these when it comes at a low conceptual cost. However, there are two complications I foresee if we allow for a non one-to-many `offsets` concept in this context:

1. If `offsets` is allowed to exist in contexts where `sizes` is disallowed, then we need a more fundamental form of schema infrastructure below one-to-many relationships that codifies this new concept (e.g. indirection array). This concept would likely just use `indices` instead of singular `offsets` as the result is identical and it would allow for a cleaner split between `sizes`/`offsets` and `indices` (which [already exists in one-to-many relationships](https://llnl-conduit.readthedocs.io/en/latest/blueprint_o2mrelation.html#protocol)).
2. If `offsets` indirection is allowed for topological connectivity arrays, why isn't it allowed for all the other Blueprint arrays? For example, couldn't a mesh's positions/fields/species also exist as non-linearly indirected entries in larger arrays? If we decide to support such broad applications of indirection, we'd need to expand our infrastructure in a concerted way.

With these two things in mind, I don't see it being feasible to have `offsets` in this particular PR (at least if we want to remain consistent with the one-to-many relationship concept), but we could add it in the future if the infrastructure is required.

Reply:
Great points, yes let's think about this more and address in the future, not in this PR.

Reply:
I've removed the `offsets` check in 509235f.